src/main/java/com/springboot/star_wars_character_popularity/app/controller/CharacterController.java

```diff
@@ -25,26 +23,26 @@ public CharacterController(CharacterService characterService){
     }
 
     @PostMapping("/save")
```

Comment:
unnecessary "/save", if you are doing PostMapping it's enough to go with root path("/api/character")